src/bin/tikv-server.rs

```diff
@@ -383,17 +388,19 @@ fn get_rocksdb_cf_option(config: &toml::Value,
     opts
 }
 
-fn get_rocksdb_default_cf_option(config: &toml::Value) -> RocksdbOptions {
+fn get_rocksdb_default_cf_option(config: &toml::Value, total_mem: u64) -> RocksdbOptions {
     // Default column family uses bloom filter.
+    let default_block_cache_size = (total_mem as f64 * 0.4) as u64;
```

Comment:
Should we define those memory ratios as a const f64 array and make sure they never exceed 1?